Makefile

```diff
@@ -34,7 +34,8 @@ upgrade: piptools $(COMMON_CONSTRAINTS_TXT)## update the requirements/*.txt file
 	sed 's/Django<2.3//g' requirements/common_constraints.txt > requirements/common_constraints.tmp
 	mv requirements/common_constraints.tmp requirements/common_constraints.txt
 
-	pip-compile --rebuild --allow-unsafe -o requirements/pip.txt requirements/pip.in
+# 	pip install -qr requirements/pip_tools.txt
```

Comment:
Why is this commented out, can it be put back now that the requirements have been updated?

Reply:
It was a mistake. Requirements are updated.